src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -27,6 +27,7 @@
 /*! The Constructor.*/
 ThrusterDynamicEffector::ThrusterDynamicEffector()
 : stepsInRamp(30)
+, fuelMass(-1.0)
```

Comment:
Why is this -1.0 instead of 0.0?

Reply:
This ensures default behavior of blow down effects not being used if a fuel tank is never attached to update the fuelMass. I couldn't make 0.0 the default behavior because that is a real possibility for the fuel tank which would make the thruster think it is out of propellant and output zero thrust, so a negative value turns off blow down instead.

Reply:
After talking, I am good with this.